datafusion-cli/tests/data/hf_store_sql.txt

```diff
@@ -0,0 +1,9 @@
+select count(*) from "hf://datasets/cais/mmlu/astronomy/dev-00000-of-00001.parquet";
```

Comment:
😮  -- tests too!

Reply:
I saw duckdb build their organizations and datasets, instead of depending on the random existing ones, to make their CI safe. It would be too early for this PR to do so... It is still in an earlier stage.